README.md

```diff
@@ -30,7 +30,70 @@ Until [WeakReference](https://ponyfoo.com/articles/weakref) is added to the Java
 
 ## Usage Example
 
-Currently our best examples are in the test folder.
+We have an example in [the example folder](./src/example).
+
+It works like this:
+
+```typescript
+// server.js
+
+import Capnode from 'capnode';
+var net = require('net');
+const through = require('through2');
+
+
+
+console.log('creating capnode client')
+const capnode = new Capnode({});
+const remote = capnode.createRemote();
+
+console.log('initializing clinet stream connection');
+var remoteStream = net.connect(5004);
+remoteStream.pipe(remote).pipe(remoteStream);
+
+async function connect() {
+    const index = await capnode.requestIndex(remote);
+    const transformed = await index.transform('beep');
+    console.log('beep => ' + transformed);
+}
+
+connect()
+.then(console.log)
+.catch(console.error)
+
+```
+
+Client then connects and can call the remote function:
+
+```typescript
+import { Stream } from "stream";
```

Comment:
Consider a comment at the top like `// client.ts` for symmetry with the `server` file above.

Reply:
Actually, I think this is the server file.